src/blocks/scratch3_operators.js

```diff
@@ -106,6 +107,11 @@ class Scratch3OperatorsBlocks {
     length (args) {
         return Cast.toString(args.STRING).length;
     }
+    
+    contains (args) {
+        return Cast.toString(args.STRING1).toLowerCase()
```

Comment:
This could be written in various ways to reduce duplication and improve readability:
```js
const format = function (string) {
    return Cast.toString(string).toLowerCase();
}

return format(args.STRING1).includes(format(args.STRING2);
```

or even ...
```js
const format = function (string) {
    return Cast.toString(string).toLowerCase();
}

const string1 = format(args.STRING1);
const string2 = format(args.STRING2);

return string1.includes(string2);
```